Makefile

```
@@ -201,6 +201,11 @@ ifeq ($(LOCAL_BUILD),true)
 	GIT_VERSION = $(shell git describe --tags --dirty --always)-dev-build
```

I think you could achieve the same by using `?=` here and the line above
```suggestion
	GIT_VERSION ?= $(shell git describe --tags --dirty --always)-dev-build
```

---

I could change the 1st one but if I change the one in LOCAL_BUILD would cause that one to never be set because of the 1st one.
Changing the 1st should be enough to do this though. Thanks.